### PR TITLE
Update Helm template values with new UI

### PR DESCRIPTION
**Pull request by APErebus**
Updates the Helm step Template values section to reflect the new UI and functionality

image

Shortcut story: [sc-92492]

### CODE REVIEW

File: dictionary-octopus.txt [hunk 1]
```diff
@@ -285,6 +285,7 @@ PKCE
 pkcs
 planetside
 podannotation
+precendence
```
[the remark this answers is not in the document]
Comment (APErebus) [reply dictionary-octopus.txt:288]: 🤦‍♂️

[the remark this answers is not in the document]
Comment (APErebus) [reply dictionary-octopus.txt]: Fixed in 7e03f5f

File: src/pages/docs/kubernetes/steps/helm.md [hunk 2]
```diff
@@ -129,15 +129,31 @@ Since it is quite common to have different versions of Helm across your deployme
...
+
+Except for **Files in the chart**, you can source template values from the same source type multiple times (e.g. you can source values from multiple different Git repositories).
+
+#### Reordering values sources
+
+The order of the template values sources dictates the prededence of how the values are applied. Sources are passed in reverse order into the `helm upgrade` command with the `-f` argument, which means that sources (and their values) higher in the list have a higher precedence and will override the same value in a source with lower prededence. See more information regarding values files prededence in the Helm [documentation](https://helm.sh/docs/chart_template_guide/values_files/).
+
+To reorder the sources, click the **Reorder** button.
+
+In the following figure, the **Key values** source value for the **drink** key will take prededence over the value for the same key in the **Inline YAML** source. The value for the **drink** key defined in the charts default `values.yaml` file will be overridden.
```
Comment (kevjt) [inline src/pages/docs/kubernetes/steps/helm.md:151]:
```suggestion
The order of the template values sources dictates the precedence of how the values are applied. Sources are passed in reverse order into the `helm upgrade` command with the `-f` argument, which means that sources (and their values) higher in the list have a higher precedence and will override the same value in a source with lower precedence. See more information regarding values files precedence in the Helm [documentation](https://helm.sh/docs/chart_template_guide/values_files/).

To reorder the sources, click the **Reorder** button.

In the following figure, the **Key values** source value for the **drink** key will take precedence over the value for the same key in the **Inline YAML** source. The value for the **drink** key defined in the charts default `values.yaml` file will be overridden.
```

Comment (kevjt) [reply src/pages/docs/kubernetes/steps/helm.md]: Love those commit messages 🤣

**Review by kevjt — APPROVED** [review]
LGTM

**Pull request merged**